src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/attributes.jl

```diff
@@ -567,7 +567,7 @@ Quotient
 ```
 """
 @attr MPolyQuoRing function coordinate_ring_of_torus(v::NormalToricVarietyType)
-    S, _ = polynomial_ring(coefficient_ring(v), vcat(coordinate_names_of_torus(v), [x*"_" for x in coordinate_names_of_torus(v)]); cached=false)
+  S, _ = polynomial_ring(coefficient_ring(v), vcat(coordinate_names_of_torus(v), [Symbol(String(x)*"_") for x in coordinate_names_of_torus(v)]); cached=false)
```

Comment:
Just FYI one could also write
```suggestion
  S, _ = polynomial_ring(coefficient_ring(v), vcat(coordinate_names_of_torus(v), [Symbol(x,"_") for x in coordinate_names_of_torus(v)]); cached=false)
```
but this has the same performance, it is just more concise.

Also, instead of `vcat` one could pass a multiple variable lists to `polynomial_argument`. But that really goes beyond this PR (but perhaps @HereAround would be interested in making such changes in the future).
```suggestion
  nams = coordinate_names_of_torus(v)
  S, _ = polynomial_ring(coefficient_ring(v), nams, [Symbol(x, "_") for x in nams]; cached=false)
```